kong/runloop/handler.lua

```diff
@@ -21,15 +22,18 @@ local certificate = require "kong.runloop.certificate"
 
 
 local kong        = kong
+local ipairs      = ipairs
 local tostring    = tostring
 local tonumber    = tonumber
 local sub         = string.sub
+local find        = string.find
 local lower       = string.lower
 local fmt         = string.format
 local sort        = table.sort
 local ngx         = ngx
 local log         = ngx.log
 local ngx_now     = ngx.now
+local rematch     = ngx.re.match
```

Comment:
Is this a conventional alias for this function? I would have expected `re_match`. (I got confused when reading further down)

Reply:
changed to `re_match`.